### PR TITLE
Sprite fixes, Oxygen Tank improvements

### LINKED ISSUE #1740: Food and drinks lose their sprite when consumed
# Description
image
Meat, icy soda and beef jerky seems to convert into a white square that goes invisible if its on the floor. 4NoRaisins seem to work properly

image
4NoRaisins and beef jerky wrapper on the same tile, one invisible

## Category
BUG

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1.Get icysoda, meat or beef jerky
2.consume it
3.notice the unintended sprite

### LINKED ISSUE #1775: Oxygen tank sprite issues
# Description
image
The oxyen tanks use their old sprite, use incorrect in-hand sprites (probably use the emergency oxygen tank in-hand sprite) and when they're on the floor they seem to be too small 

## Category
BUG & TODO

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1.Get an oxygen tank
2.Notice their old sprite 
3.Drop them on the floor
4.Notice their strangely small size sprite
5.Pick it up with your hand
6.Notice the emergency oxygen tank sprite nstead

### PR DESCRIPTION
**Pull request by fomalsd**
### Purpose
Fixes #1740
Fixes #1775 
Bullet shells, window cracks and food trash is back!
Internals work with gas containers in following slots: `"leftHand","rightHand","storage01","storage02","suitStorage"`
Big tank now has enough juice
Modified a bunch of prefabs to use `/Icons/` instead of `/Textures/`
pic

### Open Questions and Pre-Merge TODOs

- [x]  This fix is tested on the same branch it is PR'ed to.
- [x]  I correctly commented my code
- [x]  My code is indented with tabs and not spaces
- [x]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [x]  This PR does not bring up any new compile errors
- [x]  This PR has been tested in editor
- [ ]  This PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)

### Notes:
I didn't modify oxygen tank inhands;
Atmos tanks still use `/Textures/`. All lockers and crates, too

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/GasContainer.cs [hunk 1]
```diff
@@ -69,5 +69,10 @@ public void UpdateGasMix()
 		{
 			GasMix = GasMix.FromTemperature(Gases, Temperature, Volume);
 		}
+
+		/// <summary>
+		/// Slots that should be checked for gas containers
+		/// </summary>
+		public static readonly string[] GasSlots = {"leftHand","rightHand","storage01","storage02","suitStorage"};
```
Comment (Jibbajabbafic) [inline UnityProject/Assets/Scripts/Objects/GasContainer.cs]: Please add the belt and back slots too for when those features get added in future

File: UnityProject/Assets/Scripts/Equipment/Equipment.cs [hunk 2]
```diff
@@ -27,11 +29,32 @@ private void Start()
...
+
+		var slots = new List<InventorySlot>();
+		foreach ( var gasSlot in GasContainer.GasSlots )
+		{
+			if ( playerInventoryDic.ContainsKey( gasSlot ) )
+			{
+				slots.Add( playerInventoryDic[gasSlot] );
+			}
+		}
+
```
Comment (Jibbajabbafic) [inline UnityProject/Assets/Scripts/Equipment/Equipment.cs:48]: You can simplify this whole function by replacing it with this:
```suggestion
		var slots = new List<InventorySlot>();
		foreach (var slot in playerInventory)
		{
			if (GasContainer.GasSlots.Contains(slot.SlotName))
			{
				slots.Add(slot);
			}
		}

```

No need to create a new dictionary

**Comment by Aranclanos** [comment]
the in-hand sprite of the large oxygen tank should look like this
image

**Comment by Jibbajabbafic** [comment]
Foma's done it 👍 
image

**Review by Jibbajabbafic — CHANGES_REQUESTED** [review]
Good PR but needs some small changes IMO

Also found a couple other issues which I'll write up

**Comment by Aranclanos** [comment]
image
I mean these ones

**Comment by fomalsd** [comment]
@Aranclanos I looked through dm/dmi jsons and inhands seemed to be the same for those. at least the ones we have, those are from may 2017

**Comment by Jibbajabbafic** [comment]
Ah indeed the hands are wrong, good spot @Aranclanos 
image

File: UnityProject/Assets/Scripts/Equipment/Equipment.cs (hunk 2 above)
Comment (fomalsd) [reply UnityProject/Assets/Scripts/Equipment/Equipment.cs:48]: yeah, good catch

**Comment by fomalsd** [comment]
@Jibbajabbafic fixed, please check it out as I did it without editor or IDE

**Review by Jibbajabbafic — APPROVED** [review]
Looks good, thanks.

Tested in game and the sprites are correct now.

**Pull request merged**